Controller/AdminSecurityController.php

```diff
@@ -12,29 +12,29 @@
 namespace Sonata\UserBundle\Controller;
 
 use FOS\UserBundle\Model\UserInterface;
-use Symfony\Component\DependencyInjection\ContainerAware;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\RedirectResponse;
```

Comment:
having use statement for phpdoc comments makes us see better what this class depends on, even if its about return values that have no type hint in php 5. and it helps IDEs to understand the code and autocomplete. +1 to keep it that way.